### PR TITLE
Merge 3.x into 4 (for review only)

### PR DESCRIPTION
**Pull request by diosmosis**

### CODE REVIEW

File: import_logs.py [hunk 1]
```diff
@@ -436,20 +473,27 @@ def get(self, key):
     r'\s+"(?P<referrer>.*?)"\s+"(?P<user_agent>.*?)"'
 )
 
+# We need to double escape the word-boundry to work correctly, because https://docs.python.org/2/reference/lexical_analysis.html#string-literals
```
Comment (sgiehl) [inline import_logs.py:476]: Guess it's the same for Python 3 ? Maybe we can update the link to point to Python 3 docs instead then...

**Review by sgiehl — APPROVED** [review]
Looks good otherwise

File: import_logs.py (hunk 1 above)
Comment (diosmosis) [reply import_logs.py:476]: Actually I removed this bit for python 3, will remove the comment as well.

**Comment by diosmosis** [comment]
merged manually

**Pull request closed without merging**